ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-commands.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ "${BASELINE_CAPABILITY_SET}" == "" ]]; then
+  echo "This step is not required when BASELINE_CAPABILITY_SET is not set"
+  exit 0
+fi
+
+function getVersion() {
+  local release_image=""
+  if [ -n "${RELEASE_IMAGE_INITIAL-}" ]; then
+    release_image=${RELEASE_IMAGE_INITIAL}
+  elif [ -n "${RELEASE_IMAGE_LATEST-}" ]; then
+    release_image=${RELEASE_IMAGE_LATEST}
+  fi
+
+  local version=""
+  if [ ${release_image} != "" ]; then
+    oc registry login
+    version=$(oc adm release info ${release_image} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
+  fi
+  echo "${version}"
+}
+
+if [ -f "${SHARED_DIR}/kubeconfig" ] ; then
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
+fi
```

Comment:
As following steps require to access cluster, I would suggest to exit directly if `${SHARED_DIR}/kubeconfig` not found.